certbot/plugins/manual.py

```diff
@@ -60,10 +93,24 @@ class Authenticator(common.Plugin):
 "import BaseHTTPServer, SimpleHTTPServer; \\
 s = BaseHTTPServer.HTTPServer(('', {port}), SimpleHTTPServer.SimpleHTTPRequestHandler); \\
 s.serve_forever()" """
+    _TLSSNI_INSTRUCTIONS = """\
+Configure the service listening on port {port} to serve the certificate
+{cert}
+along with the secret key
+{key}
+
+If you don't have a server running on port {port}, you can run the
+following command (as root):
+
+openssl s_server -accept {port} -cert {cert} -key {key} -www
```

Comment:
In #4804, we decided to remove instructions for how to do it without a web server as they're usually visual noise and they're hard to get right. For example, as you pointed out above, this doesn't work if multiple challenges need to be performed on the same machine and port. (We had similar problems with the HTTP as described in #1032).

With this in mind, I think instructions for using `openssl s_server` here should be removed.